pxe-formula/pxe-formula.spec

```
@@ -15,16 +15,17 @@
 # Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
+
+%define fname pxe
 Name:           pxe-formula
 Version:        0.2.0
 Release:        0
 Summary:        Formula for atftpd server on POS branchserver
-License:        GPL-2.0
+License:        GPL-2.0-only
```

I think it should be:

```suggestion
License:        GPL-2.0+
```
the  header uses:
~~~
the Free Software Foundation, either version 2 of the License, or
(at your option) any later version.
~~~